src/components/App/App.js

```diff
@@ -6,7 +6,7 @@ import 'perseids-react-components/build/css/index.css';
 import EditXml from '../EditXml';
 import InputXml from '../InputXml';
 
-import './App.css';
+import './App.module.css';
```

Comment:
Let's get rid of this line entirely and delete `App.module.css`. It looks like it's an empty file.